maestro-cli/src/main/java/maestro/cli/command/BugReportCommand.kt

```diff
@@ -14,6 +13,9 @@ import java.util.concurrent.Callable
 )
 class BugReportCommand : Callable<Int> {
 
+    @CommandLine.Mixin
+    var disableANSIMixin: DisableAnsiMixin? = null
```

Comment:
Does this need to go on every subcommand or can it just be at the app level?

Reply:
This needs to be at every subcommand. Picocli does not support setting parameters on every subcommand without changing the subcommand definition. By using a mixin, the impact is minimal.

Reply:
I think what we want is the ability to do `maestro --no-ansi <anysubcommand>` as opposed to `maestro <anysubcommand> --no-ansi`. Do we still need to add to every subcommand in this case?

Reply:
In that case no, but users will be confused that `maestro test --no-ansi` does not work

Reply:
I think this tradeoff is fine since we have this issue with other options as well. I agree we should consider fixing this, but i think the proper approach would be to add a subclass that all our commands inherit from so all our base option can benefit from this. For now I don't think it's worth the duplication of code to fix for only --no-ansi as this problem already exists for other options

Reply:
I know I will be confused. I know because we have the same argument / command requirement with our internal cli tools.

Reply:
Yep agree we should address - but I think we should fix this holistically instead of just for a single option

Reply:
And I believe it makes sense to address that as a follow up